src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -17,7 +17,10 @@
 using System.Diagnostics.Contracts;
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks.Dataflow.Internal;
+
+#if !THREADING
```

Comment:
I wanted to keep it consistent with `CONCURRENT_COLLECTIONS` and `FEATURE_TRACING` in that the constant describes something available in the newer versions of .Net Standard.

The current name is generic, but I can't think of anything better that means "has `Timer`, `Thread` and `ThreadPool`".